store/interfaces.json

```diff
@@ -1,6 +1,8 @@
 {
   "amp-store": {
     "component": "*",
+    "before": ["header.full"],
```

Comment:
Header doesn't work properly in amp page because `react-intl` can't transform messages

Reply:
doesn't it transform them during the server render?